src/types/requests/HttpChargingStationRequest.ts

```diff
@@ -53,10 +53,25 @@ export interface HttpChargingStationParamsUpdateRequest {
   public: boolean;
   excludeFromSmartCharging: boolean;
   forceInactive: boolean;
+  manualConfiguration: boolean;
   siteAreaID: string;
   coordinates: number[];
+  chargePoints: {
```

Comment:
Yes you are right. I did this because it was also done like this in the connector, but since we are using every property of the charge point we can also use the type. 
It should be an array already. 
```
efficiency: number;
connectorIDs: number[]}[];
```